packages/components/src/components/info-card.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import { Meta, StoryObj } from '@storybook/react';
+import { hiveThemeDecorator } from '../../../../.storybook/hive-theme-decorator';
+import { AccountBox } from './icons';
+import { InfoCard, InfoCardProps } from './info-card';
+
+export default {
+  title: 'Components/InfoCard',
+  component: InfoCard,
+  decorators: [hiveThemeDecorator],
+} satisfies Meta<InfoCardProps>;
+
+export const Default: StoryObj<InfoCardProps> = {
+  args: {
```

Comment:
```suggestion
export const Default: StoryObj<InfoCardProps> = {
  name: 'InfoCard',
  args: {
```
like in Stud 
<img width="270" alt="image" src="https://github.com/user-attachments/assets/317e8f84-02d3-4e88-8db6-a0c12da5c85b">

to not have a list? or do you plan to add more stories here?